pkgs/desktops/xfce4-13/exo/default.nix

```diff
@@ -9,7 +9,7 @@ mkXfceDerivation rec {
   sha256 = "0zxv7cx1xbjls7q2blv8ir9zwzyq7r189n6q35jwasns7rxj256v";
 
   nativeBuildInputs = [ libxslt perlPackages.URI ];
-  buildInputs = [ gtk2 gtk3 libxfce4ui libxfce4util ];
+  buildInputs = [ gtk2 gtk3 libxfce4ui libxfce4util autoconf automake intltool libtool gtk-doc ];
```

Comment:
See `xfce4-dev-tools` and https://github.com/NixOS/nixpkgs/issues/37991#issuecomment-380621853.